### ISSUE TITLE
Please update to latest jslint version

### ISSUE THREAD

**Issue by lo1tuma** [stephenmathieson/grunt-jslint#30]
grunt-jslint does’t report all errors of missing space between function keyword and parenthesis of an anonymous function.

Example: 

``` javascript
(function () {
    'use strict';

    module.exports = {
        start: function () {
            this.router.on('action:start', function() { this.set({loading: false}); });
        }
    };

}());
```

The callback function violates clearly jslint rules but this error doesn’t get reported.

This issue is probably fixed by ba6642aa85 commit of jslint
